workers/queue_backend/pg_queue/client.py

```
@@ -0,0 +1,475 @@
+
+        Two callers: the lease-renewal thread (keeps a live claim's ``vt`` in the
+        future so the reaper never re-arms it), and the consumer deferring a poison
+        message whose terminal-ERROR mark could not be confirmed (backend down), so
+        the drop never races a dead backend and the payload isn't discarded.
+        """
+        if vt_seconds <= 0:
+            raise ValueError(f"vt_seconds must be positive, got {vt_seconds}")
+        with self._cursor() as cur:
+            cur.execute(
+                f"UPDATE {qualified('pg_queue_message')} "
+                "SET vt = now() + make_interval(secs => %s) WHERE msg_id = %s",
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)